integration_tests/models/_integration_tests__models.yml

```diff
@@ -29,6 +29,16 @@ models:
       tags: claims
       materialized: table
 
+  - name: provider_attribution
+    description: Created from seed or source data depending on test_data_override variable.
+    config:
+      schema: |
+        {%- if var('tuva_schema_prefix',None) != None -%}_{{var('tuva_schema_prefix')}}input_layer
+        {% else %}input_layer{%- endif -%}
+      tags: claims
```

Comment:
Perhaps have the tag be attribution? That way I could run this without all the other claims input

Reply:
I agree.